### PR TITLE
Fixbug/springmvn-annotation-4.x-plugin, witness class does not exist in some versions

### LINKED ISSUE #5790: ServletContextPropertyPlaceholderConfigurer is not found in some new version for springframework 4
#### skywalking-api.log 
```
org.apache.skywalking.apm.plugin.spring.mvc.v4.define.RestControllerForLowVersionInstrumentation is not working. 
Because witness class org.springframework.web.context.support.ServletContextPropertyPlaceholderConfigurer is not existed. 
```

#### ServletContextPropertyPlaceholderConfigurer removed in some new version for springframework 4

1. [cannot-find-class-org-springframework-web-context-support-servletcontextproperty](https://stackoverflow.com/questions/60722169/cannot-find-class-org-springframework-web-context-support-servletcontextproperty)

1. [cannot find in github too](https://github.com/spring-projects/spring-framework/find/v4.3.24.RELEASE)

#### in code check ServletContextPropertyPlaceholderConfigurer  for springmvc 4
https://github.com/apache/skywalking/blob/9042ce6fe225a5c68fd00185e7dec9dff14b9eee/apm-sniffer/apm-sdk-plugin/spring-plugins/mvc-annotation-4.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/spring/mvc/v4/define/RestControllerForLowVersionInstrumentation.java#L28

any way to solve this problem?

thx

### PR DESCRIPTION
**Pull request by LiWenGu**
- [x] Closes #5790
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

File: changes/changes-8.4.0.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ Release Notes.
 * Support DNS periodic resolving mechanism to update backend service.
 * Support config `agent.trace.ignore_path` can be changed in the runtime.
 * Added support for transmitting logback 1.x and log4j 2.x formatted & un-formatted messages via gPRC
+* Fix bug that springmvn-annotation-4.x-plugin, witness class does not exist in some versions.
```
Comment (wu-sheng) [inline changes/changes-8.4.0.md:44]: We are on 8.5.0 iteration, changelog file in the root folder.

**Comment by wu-sheng** [comment]
> Fixbug/springmvn-annotation-4.x-plugin, witness class does not exist in some versions

You should add these versions in the 4.x/5.x test cases. https://github.com/apache/skywalking/blob/master/test/plugin/scenarios/spring-4.3.x-scenario/support-version.list

**Comment by wu-sheng** [comment]
@LiWenGu Are you still following?

**Review by wu-sheng — APPROVED** [review]
Once plugun tests passed. This is good to me.

**Comment by LiWenGu** [comment]
> Once plugun tests passed. This is good to me.

better!

**Pull request merged**